internal/cli/clients/client.go

```diff
@@ -734,6 +741,74 @@ func (c *EpinioClient) AppUpdate(appName string, instances int) error {
 	return nil
 }
 
+// AppLogs streams the logs of all the application instances, in the targeted org
+func (c *EpinioClient) AppLogs(appName string, follow bool) error {
+	log := c.Log.WithName("Apps").WithValues("Organization", c.Config.Org, "Application", appName)
+	log.Info("start")
+	defer log.Info("return")
+	details := log.V(1) // NOTE: Increment of level, not absolute.
+
+	c.ui.Note().
+		WithStringValue("Organization", c.Config.Org).
+		WithStringValue("Application", appName).
+		Msg("Streaming application logs")
+
+	details.Info("application logs")
+
+	interrupt := make(chan os.Signal, 1)
+	signal.Notify(interrupt, os.Interrupt)
+
+	var urlArgs = []string{}
+	urlArgs = append(urlArgs, fmt.Sprintf("follow=%t", follow))
+
+	headers := http.Header{
+		"Authorization": {"Basic " + base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf("%s:%s", c.Config.User, c.Config.Password)))},
+	}
+
+	webSocketConn, _, err := websocket.DefaultDialer.Dial(fmt.Sprintf("%s/%s?%s", c.wsServerURL, api.Routes.Path("AppLogs", c.Config.Org, appName), strings.Join(urlArgs, "&")), headers)
+	if err != nil {
+		return err
+	}
+
+	done := make(chan bool)
+	errorChan := make(chan error, 1)
+
+	go func() {
+		for {
+			_, message, err := webSocketConn.ReadMessage()
+			if err != nil {
+				if websocket.IsCloseError(err, websocket.CloseNormalClosure) {
+					c.ui.Normal().Msg("-- Logs End --")
+					webSocketConn.Close()
+					done <- true
+					return
+				}
+				errorChan <- err
+				return
+			}
+			fmt.Println(string(message))
```

Comment:
I'm not sure we read line by line (https://github.com/epinio/epinio/pull/402/files#diff-168b4a07f391b22def39fd870b0373811deb114cd513793f9cf5a3182f8452ceR249), maybe we shouldn't print new line characters?